SU-HSDO/suhumsci
# SHS-5582: Bug: Issue with Add/Select Media form field

# READY FOR REVIEW

## Summary
Fix the "Sort by" label and form field that has incorrect styling when site editor uses the Media Library in CKEditor

## Need Review By (Date)
04/03

## Urgency
low

## Steps to Test
1. Create a Flexible Page on Colorful and Traditional
2. Select the Media Library in CKEditor and confirm the "Sort by" label and form field looks like this
3. Confirm any select/combox still looks as expected in other forms (Views for example)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
